tools/bumper/cnao_repo_commands_test.go

```diff
@@ -28,6 +28,11 @@ var _ = Describe("Testing internal git CNAO Repo", func() {
 		githubApi = newFakeGithubApi(repoDir)
 
 		gitCnaoRepo = newFakeGitCnaoRepo(githubApi, repoDir, &component{}, expectedTagCommitMap)
+
+		DeferCleanup(func() {
+			Expect(os.RemoveAll(gitCnaoRepo.gitRepo.localDir)).To(Succeed())
```

Comment:
Do you need both of them? Isn't the first dir under the 2nd?

Reply:
I though about keeping them both but maybe it's an overkill.